packages/toolkit/package.json

```diff
@@ -46,7 +46,8 @@
     "fs-extra": "^9.0.0",
     "get-folder-size": "^2.0.1",
     "go-platform": "^1.0.0",
-    "ipfs-http-client": "^40.1.0",
+    "ipfs-http-client": "^41.0.1",
```

Comment:
Wondering—do you think we should investigate why the new versions were failing, or should we just soft pin to this version using `~`?

Reply:
I think it’s definitely worth investigating further. `ipfs-http-client` is a relatively popular and well-maintained library so this could point to a problem in our code.